packages/loaders/stories/5-Skeleton.stories.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import styled from 'styled-components';
+import { Skeleton, ISkeletonProps } from '@zendeskgarden/react-loaders';
+import { XXXL, MD } from '@zendeskgarden/react-typography';
+import { PALETTE } from '@zendeskgarden/react-theming';
+
+export default {
+  title: 'Components/Loaders/Skeleton',
+  component: Skeleton
+} as Meta;
+
+const StyledExampleWrapper = styled.div<Partial<ISkeletonProps>>`
+  background-color: ${props => (props.isLight ? PALETTE.kale[700] : PALETTE.white)};
+  padding: 18px;
```

Comment:
Should these use `theme`, `${props => props.theme.space.base * 4};`?

Reply:
I've updated this to use one of the named spacing constants.